api/lib/reducers/cosmosV2-reducers.js

```diff
@@ -337,7 +346,17 @@ function transactionReducerV2(network, transaction, reducers) {
             ? transaction.logs[index].log || transaction.logs[0] // failing txs show the first logs
             : transaction.logs[0].log || ''
           : JSON.parse(JSON.stringify(transaction.raw_log)).message,
-      involvedAddresses: uniq(reducers.extractInvolvedAddresses(transaction))
+      involvedAddresses: uniq(reducers.extractInvolvedAddresses(transaction)),
```

Comment:
instead this should be fixed to have the involved addresses for this message only. you can use the code you have here

Reply:
ok